### PR TITLE
[SPARK-9267][Core] Retire stringify(Partial)?Value from Accumulators

### PR DESCRIPTION
**Pull request by huitseeker**
cc @srowen

### CODE REVIEW

**Comment by srowen** [comment]
LGTM

**Comment by huitseeker** [comment]
#1203 failure is unrelated:
```
Finished test(python2.6): pyspark.sql.column (12s)
..E...........Exception SystemExit: SystemExit(1,) in <module 'threading' from '/usr/lib64/python2.6/threading.pyc'> ignored
................................................cc: no input files
cc: no input files
cc: no input files
cc: no input files
cc: no input files
cc: no input files
cc: no input files
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.6/threading.py", line 532, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.6/threading.py", line 484, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/jenkins/workspace/NewSparkPullRequestBuilder/python/pyspark/rdd.py", line 719, in pipe_objs
    out.write(s.encode('utf-8'))
IOError: [Errno 32] Broken pipe

cc: no input files
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.6/threading.py", line 532, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.6/threading.py", line 484, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/jenkins/workspace/NewSparkPullRequestBuilder/python/pyspark/rdd.py", line 719, in pipe_objs
    out.write(s.encode('utf-8'))
IOError: [Errno 32] Broken pipe
```

**Comment by sarutak** [comment]
retest this please.

**Comment by sarutak** [comment]
retest this please.

**Comment by srowen** [comment]
Wow, 5 failures, 4 different causes. Unlucky, but also, seems like a particularly bad state for Jenkins/tests at the moment.

File: core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala [hunk 1]
```diff
@@ -896,11 +896,9 @@ class DAGScheduler(
           // To avoid UI cruft, ignore cases where value wasn't updated
           if (acc.name.isDefined && partialValue != acc.zero) {
             val name = acc.name.get
-            val stringPartialValue = Accumulators.stringifyPartialValue(partialValue)
-            val stringValue = Accumulators.stringifyValue(acc.value)
-            stage.latestInfo.accumulables(id) = AccumulableInfo(id, name, stringValue)
+            stage.latestInfo.accumulables(id) = AccumulableInfo(id, name, s"${acc.value}")
             event.taskInfo.accumulables +=
-              AccumulableInfo(id, name, Some(stringPartialValue), stringValue)
+              AccumulableInfo(id, name, Some(s"partialValue"), s"${acc.value}")
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala]: eh, just noticed this, but doesn't this need to be `s"$partialValue"`?

Comment (huitseeker) [inline core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala]: >_< :hurtrealbad:

**Comment by srowen** [comment]
I'm going to merge this, since the last Python test failures can't be related to this change, and the tests had previously passed, and this has been through so many flaky test failures already.

**Pull request closed without merging**